server/tls/index.js

```diff
@@ -1,56 +1,35 @@
 const grpc = require('grpc');
-const { readFileSync } = require('fs');
+const { readCredsFromCertFiles } = require('./read-creds-from-cert-files');
+const { readCredsFromYml } = require('./read-creds-from-yml-file');
 
-const caPath = process.env.TEMPORAL_TLS_CA_PATH;
 const keyPath = process.env.TEMPORAL_TLS_KEY_PATH;
 const certPath = process.env.TEMPORAL_TLS_CERT_PATH;
+const caPath = process.env.TEMPORAL_TLS_CA_PATH;
 const serverName = process.env.TEMPORAL_TLS_SERVER_NAME;
 const verifyHost = [true, 'true', undefined].includes(
   process.env.TEMPORAL_TLS_ENABLE_HOST_VERIFICATION
 );
+const configPath = process.env.TEMPORAL_TLS_CONFIG_PATH;
```

Comment:
[Nit] can we call this tlsYmlPath or something? configPath is too generic of a name.

Reply:
yes, sounds good. Was thinking about Yml Path as the second option, agreed